src/cse-machine/interpreter.ts

```diff
@@ -584,14 +584,14 @@ const cmdEvaluators: { [type: string]: CmdEvaluator } = {
     // Refer to Source §3 specifications https://docs.sourceacademy.org/source_3.pdf
     if (init.type === 'VariableDeclaration' && init.kind === 'let') {
       const id = init.declarations[0].id as es.Identifier
-      const valueExpression = init.declarations[0].init!
+      // const valueExpression = init.declarations[0].init!
```

Comment:
Let's try to not leave commented code around. If it's unused, remove it entirely, and if it's only unused for now, leave another comment explaining why.